### PR TITLE
Update php to 8.2 and openswoole to 22.0 

### PR DESCRIPTION
**Pull request by Xerkus**
Supersedes #39

Uses branch in this repository to trigger build on platform.sh

### CODE REVIEW

**Review by Ocramius — APPROVED** [review]
LGTM if this serves actual HTTP responses somewhere

**Comment by Xerkus** [comment]
Fails to build swoole on platform.sh with weird error. Needs further investigation.

**Comment by Xerkus** [comment]
Blocked by https://github.com/mezzio/mezzio-swoole/issues/110

**Pull request merged**
